pgd_search/tests.py

```diff
@@ -768,3 +768,114 @@ def test_sidechain_statistics_present(self):
+        comp_option.click()
+
+        #Conformation
+        conformation = self.driver.find_element_by_id("id_ss_choices_list_col_2")
+        conf_option = conformation.find_elements_by_tag_name('li')[0]
+        conf_option.click()
+
+        conformation_phi = self.driver.find_element_by_id("id_phi_2")
+        conformation_phi.clear()
+        conformation_phi.send_keys("<=-85,>=85")
+
+        conformation_psi = self.driver.find_element_by_id("id_psi_2")
+        conformation_psi.clear()
+        conformation_psi.send_keys("<=-80,>=80")
+
+        conformation_omega = self.driver.find_element_by_id("id_ome_2")
+        conformation_omega.clear()
+        conformation_omega.send_keys("<=-75,>=75")
+
+        #Mobility
+        self.driver.find_element_by_id("mobility_header").click()
+        mobility = self.driver.find_element_by_id("id_bm_2")
+        mobility.clear()
+        mobility.send_keys("<35")
+
+        #Angles
+        self.driver.find_element_by_id("angles_header").click()
+        angles = self.driver.find_element_by_id("id_a1_2")
+        angles.send_keys("30")
+
+        #Lengths
+        self.driver.find_element_by_id("lengths_header").click()
+        lengths = self.driver.find_element_by_id("id_L1_2")
+        lengths.send_keys("25")
+
+        #XAngles
+        self.driver.find_element_by_id("chi_header").click()
+        xangles = self.driver.find_element_by_id("id_chi1_2")
+        xangles.send_keys("20")
+
+        #Hackish way to do it, but there doesn't seem to be any other
+        #common ways to do it.
+        residues = self.driver.find_element_by_id("id_residues")
+        for option in residues.find_elements_by_tag_name('option'):
+            if option.text == "3":
+                option.click()
+
+        for option in residues.find_elements_by_tag_name('option'):
+            if option.text == "4":
+                option.click()
+
+        #Composition
+        self.assertEquals(comp_option.get_attribute("class"), " ")
+
+        #Conformation
+        self.assertEquals(conf_option.get_attribute("class"), " ")
+        self.assertEquals(conformation_phi.get_attribute("value"), "")
+        self.assertEquals(conformation_psi.get_attribute("value"), "")
+        self.assertEquals(conformation_omega.get_attribute("value"), "<=-90,>=90")
+
+        #Mobility
+        self.assertEquals(mobility.get_attribute("value"), "<25")
+
+        #Angles
+        self.assertEquals(angles.get_attribute("value"), "")
+
+        #Lengths
+        self.assertEquals(lengths.get_attribute("value"), "")
+
+        #XAngles
+        self.assertEquals(xangles.get_attribute("value"), "")
+
+        #Clicking the submit button
+        self.driver.find_element_by_xpath("//input[@type='submit']").click()
+
+        #ugly hack. http://goo.gl/yOieEo
+        time.sleep(5)
+
+        #Waiting to click for the page to load
+        element = WebDriverWait(self.driver, 10).until(
```

Comment:
I commented out the sleep line and bumped this timeout up to 60 and the test passed ten times in a row.